api/repositories.py

```diff
@@ -39,6 +39,18 @@ class UserRepository(Repository):
     def __init__(self, mongo_collection, repository_collection):
         Repository.__init__(self, mongo_collection, repository_collection)
 
+    def update_user_account(self, user_id, name, password):
+        user = self.get_user_by_id(user_id)
+        password_hash = bcrypt.hashpw(password.encode('utf-8'), bcrypt.gensalt()).decode('utf-8')
```

Comment:
So the function will return False when the name and password are both None? Otherwise it will update.